gateway/handlers/forwarding_proxy.go

```diff
@@ -139,6 +139,10 @@ func copyHeaders(destination http.Header, source *http.Header) {
 		copy(vClone, v)
 		(destination)[k] = vClone
 	}
+
+	for _, h := range hopHeaders {
```

Comment:
@stefanprodan PTAL

Reply:
Removed here -> https://github.com/openfaas/faas/pull/1146

Reply:
@LucasRoesler we added this code elsewhere please can you remove this code if duplicated?